src/devices/yokis.ts

```diff
@@ -0,0 +1,2642 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* TODO : remove eslint-disable when dev is done */
```

Comment:
TODO: remove

Reply:
Done - removed unused vars, and commented others if i need them in the future.